lavamoat/browserify/main/policy.json

```diff
@@ -895,14 +894,14 @@
         "@metamask/assets-controllers>@metamask/base-controller": true,
         "@metamask/assets-controllers>@metamask/controller-utils": true,
         "@metamask/assets-controllers>@metamask/polling-controller": true,
+        "@metamask/assets-controllers>@metamask/utils": true,
         "@metamask/assets-controllers>cockatiel": true,
         "@metamask/assets-controllers>multiformats": true,
         "@metamask/contract-metadata": true,
         "@metamask/eth-query": true,
         "@metamask/metamask-eth-abis": true,
         "@metamask/name-controller>async-mutex": true,
         "@metamask/rpc-errors": true,
-        "@metamask/utils": true,
```

Comment:
"@metamask/utils" turning into "@metamask/assets-controllers>@metamask/utils" means assets-controller is using a different version of utils and they're not getting deduplicated by yarn. 
- is that expected?
- is that desired?
-

Reply:
You right, thanks for pointing that out 🙏  
`assets-controllers` is using v9.0.0 of `@metamask/utils` while extension is using v8.4.0;
Looking at the [changelog](https://github.com/MetaMask/utils/blob/main/CHANGELOG.md), the only breaking change mentioned there is this 
`BREAKING: The return types of functions getChecksumAddress, numberToHex, bigIntToHex are narrowed from string to Hex`
Which i do not has affected the assets-controllers upgrade. I also haven't seen any weird errors related to this while testing locally;

Reply:
If you already did the testing, you might update utils to 9 too. Up to you.
I don't see anything else interesting in the policy diff.

Reply:
Thank you @naugtur  for checking the policies 🙏 
I think i will keep the utils version as it is, since its not affection assets flow,